pkg/jsonutil/conversion_test.go

```diff
@@ -111,7 +112,13 @@ func TestNoErrorOnUnknownAttribute(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "6500", proto.GetId())
 
-	err = JSONBytesToProto([]byte(json), &proto)
+	jsonBytes := []byte(json)
+	err = JSONBytesToProto(jsonBytes, &proto)
+
+	assert.NoError(t, err)
+	assert.Equal(t, "6500", proto.GetId())
+
+	err = JSONReaderToProto(bytes.NewReader(jsonBytes), &proto)
 
 	assert.NoError(t, err)
 	assert.Equal(t, "6500", proto.GetId())
```

Comment:
I suggest to unbreak lines to highlight different cases better
```suggestion
	jsonBytes := []byte(json)

	err = JSONBytesToProto(jsonBytes, &proto)
	assert.NoError(t, err)
	assert.Equal(t, "6500", proto.GetId())

	err = JSONReaderToProto(bytes.NewReader(jsonBytes), &proto)
	assert.NoError(t, err)
	assert.Equal(t, "6500", proto.GetId())
```